tests/v2/test_conversion.py

```diff
@@ -36,10 +36,10 @@ def test_benchmark_collection(problem_id):
     logging.basicConfig(level=logging.DEBUG)
 
     if problem_id == "Froehlich_CellSystems2018":
+        # this is mostly about 6M sympifications in the condition table
         pytest.skip("Too slow. Re-enable once we are faster.")
```

Comment:
Not sure but maybe caching helps, e.g. https://github.com/AMICI-dev/AMICI/pull/1672/files

Reply:
I had the feeling that caching *is* the problem here (inside sympy), but that needs further investigation.